setuptools/tests/test_config.py

```diff
@@ -802,6 +802,40 @@ def test_entry_points(self, tmpdir):
         with get_dist(tmpdir) as dist:
             assert dist.entry_points == expected
 
+    def test_case_sensitive_entry_points(self, tmpdir):
+        _, config = fake_env(
+            tmpdir,
+            '[options.entry_points]\n'
+            'GROUP1 = point1 = pack.module:func, '
+            '.point2 = pack.module2:func_rest [rest]\n'
+            'group2 = point3 = pack.module:func2\n'
+        )
+
+        with get_dist(tmpdir) as dist:
+            assert dist.entry_points == {
+                'GROUP1': [
+                    'point1 = pack.module:func',
+                    '.point2 = pack.module2:func_rest [rest]',
+                ],
+                'group2': ['point3 = pack.module:func2']
+            }
+
+        expected = (
+            '[blogtool.parsers]\n'
+            '.rst = some.nested.module:SomeClass.some_classmethod[reST]\n'
+        )
+
+        tmpdir.join('entry_points').write(expected)
+
+        # From file.
+        config.write(
+            '[options]\n'
+            'entry_points = file: entry_points\n'
+        )
+
+        with get_dist(tmpdir) as dist:
+            assert dist.entry_points == expected
```

Comment:
I'm slightly surprised by this behavior. I'm not sure `file:` support was intended for `options.entry_points`, although I agree it would be nice if it were supported. What's more surprising and concerning is that `dist.entry_points` is a different format. I see now that this test is modeled after `test_entry_points`. Since that test already essentially covers this case, I would recommend omitting the second half. WDYT?

Reply:
I was also unsure about that section of the test. I thought to include it in case the case-sensitive fix broke something that `test_entry_points` was supposed to pass. Although, it does appear to be passing regardless. I'll go ahead and remove it as I'm uncertain about its behaviour.